pdr_backend/util/predictoor_stats.py

```diff
@@ -0,0 +1,128 @@
+from typing import List, Dict, Tuple, Union
+from pdr_backend.models.prediction import Prediction
+
+PredictionDetail = Tuple[str, str, str]
+PredictoorStats = Dict[str, Union[str, float, int, List[PredictionDetail]]]
+PairTimeframeStats = Dict[str, Union[str, float, int]]
+StatisticsResult = Dict[
+    str, Union[float, List[PredictoorStats], List[PairTimeframeStats]]
+]
+
+
+def get_pure_stats(all_predictions: List[Prediction]):
```

Comment:
What does "pure" mean? This needs a better name.

Reply:
renamed as "aggregate_prediction_statistics"